.github/workflows/build.yml

```diff
@@ -16,13 +16,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-13, windows-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
For background, the reason why i pinned it to -13 at one point was because eto/dotnet was behaving really strangely with back then it being latest. Not sure if it's fixed now.

Reply:
macos still fails, but I don't consider it blocking.
> LLINK : warning MM0079: The recommended Xcode version for Microsoft.macOS 15.0.8319 is Xcode 16.0 or later. The current Xcode version (found in /Applications/Xcode_15.4.app/Contents/Developer) is 15.4. [/Users/runner/work/AM2RLauncher/AM2RLauncher/AM2RLauncher/AM2RLauncher.Mac/AM2RLauncher.Mac.csproj]
  		
> ILLink : unknown error IL7000: An error occurred while executing the custom linker steps. Please review the build log for more information. [/Users/runner/work/AM2RLauncher/AM2RLauncher/AM2RLauncher/AM2RLauncher.Mac/AM2RLauncher.Mac.csproj]

Reply:
@peeley you wanna try changing ~line 44 for this or keep it as is?

Reply:
Oh man, I didn't even notice that line. I was wondering why the Xcode version was behaving weirdly. I'm gonna just try and delete that, since the same command works locally on MacOS (and I don't see why that specific version of XCode is needed, instead of the latest version).

Reply:
macos-latest doesnt *have* the latest xcode enabled by default for whatever reason.
At least last that was the case when I added it. Deleting it doesnt actually do anyting because it currently doesnt already get triggered on your branch because its wrapped in an `if macos-13`, while we're on `macos-latest`

The switch there needs to be changed to 16.0 or 16.1

See here: https://github.com/actions/runner-images/blob/main/images/macos/macos-14-Readme.md#xcode

Reply:
Turns out I was misunderstanding what `macos-latest` points to - I thought it was `macos-15`, but it points to `macos-14` ([src](https://github.com/actions/runner-images?tab=readme-ov-file#available-images)). `macos-15` has the [correct XCode version](https://github.com/actions/runner-images/blob/main/images/macos/macos-15-Readme.md#installed-sdks), so I think I need to point to `macos-15` explicitly.

Reply:
macos-15 is still in beta (your source) so I'd like to not point to it explicitly.
I'd just point to macos-latest and have the xcode switch removed - by the time someone makes another pull request macos15 is hopefully Generally Available and then those tests.